hypershift-operator/controllers/hostedcluster/hostedcluster_controller_test.go

```diff
@@ -1404,35 +1404,6 @@ func TestValidateConfigAndClusterCapabilities(t *testing.T) {
 			},
 			expectedResult: errors.New(`[credentials secret for cluster doesn't have required key AZURE_CLIENT_ID, credentials secret for cluster doesn't have required key AZURE_CLIENT_SECRET, credentials secret for cluster doesn't have required key AZURE_SUBSCRIPTION_ID, credentials secret for cluster doesn't have required key AZURE_TENANT_ID]`),
 		},
-		{
```

Comment:
This would require actual Azure cloud resources setup now so I removed the test case.

Reply:
Not for this PR, but can we eventually use a mock implementation of the APIs to run the test?

Reply:
Agree, it's something we need to look into